src/cmd/tools/helm/repo_add.go

```diff
@@ -128,11 +130,15 @@ func (o *repoAddOptions) run(out io.Writer) error {
 		lockPath = o.repoFile + ".lock"
 	}
 	fileLock := flock.New(lockPath)
-	lockCtx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	lockCtx, cancel := context.WithTimeout(context.Background(), cluster.DefaultTimeout)
```

Comment:
Add context to run and pass cobra Command context to function.

Reply:
updated